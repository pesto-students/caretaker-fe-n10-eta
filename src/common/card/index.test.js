import React from "react";
import renderer from "react-test-renderer";
import CommonCard from "./index";

it("Card renders correctly", () => {
  const tree = renderer
    .create(
      <CommonCard>
        <div>Hi</div>
      </CommonCard>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
