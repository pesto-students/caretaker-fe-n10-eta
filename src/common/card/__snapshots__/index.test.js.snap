// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card renders correctly 1`] = `
<div
  className="ant-card ant-card-bordered card"
>
  <div
    className="ant-card-body"
    style={Object {}}
  >
    <div>
      Hi
    </div>
  </div>
</div>
`;
